### PR TITLE
Use public NTP server by default in compute nodes

### PR DESCRIPTION
**Pull request by pablintino**
Chrony NTP client running inside compute nodes is getting configured by default with clock.redhat.com, a NTP server that is not publicly available and, thus, not a safe default to be used for this public repository.

### CODE REVIEW

**Review by stuggi — COMMENTED** [review]
+1 from my side, lets get some others feedback before landing it

**Comment by raukadah** [comment]
Thank you for the pull request. Can we also remove DATAPLANE_CHRONY_NTP_SERVER from https://github.com/openstack-k8s-operators/install_yamls/blob/master/ci/playbooks/deploy_edpm.yaml#L37 ?

**Comment by stuggi** [comment]
since we change the default and deployments might fail, should we mention to optionally set the ntp server in the doc at https://github.com/openstack-k8s-operators/install_yamls/blame/master/README.md#L138 ?

**Comment by pablintino** [comment]
Botch comments seems reasonable. I'll push a commit removing the env var in the playbook and adding a note in the doc at that point.

**Review by raukadah — APPROVED** [review]
Thank you :-)

**Comment by bshephar** [comment]
Looks fine to me, but we might also need some other CI related files to account for this. Best to check with CI folks to ensure this isn't going to cause pain somewhere else down the line.

**Comment by cjeanner** [comment]
IMHO it's better to get a default allowing standard upstream work. We can then override it when working downstream. Also, as a general thought, we shouldn't rely on anything "downstream specific" in a public repository.

I'd go with a "merge it" directly, but if we're seeking more feedback... We may want to wait until 2023-04-18 EOD UTC (that is tomorrow), and merge it if no one is complaining.

**Comment by raukadah** [comment]
Approving it. If further suggestions comes, we can update it via followup pull request.

**Pull request merged**
